src/app/debug/filter-side-drawer/filter-side-drawer.component.html

```diff
@@ -0,0 +1,29 @@
+<div @removeTrigger class="filter-vertical-panel" data-cy-debug="filter-side-drawer" *ngIf="shouldShowFilter">
```

Comment:
Please replace this `*ngIf` with `@if`, just like you do with the `@for`

Reply:
added the change